build.psm1

```diff
@@ -2210,7 +2210,7 @@ function Start-CrossGen {
     $crossGenRequiredAssemblies = @("mscorlib.dll", "System.Private.CoreLib.dll")
```

Comment:
@SteveL-MSFT, your last commit had 4 failures in `PowerShell-CI-windows`
Verify approved aliases list.All approved aliases present (no new aliases added, no aliases removed)
```powershell
Expected $null or empty, but got @(@{InputObject=ogv -> Out-GridView; SideIndicator=<=}, @{InputObject=shcm -> Show-Command; SideIndicator=<=}).
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\DefaultCommands.Tests.ps1: line 553
553:         $result | Should -BeNullOrEmpty
```
Verify approved aliases list.All approved aliases have the correct 'ReadOnly' option
```powershell
Expected $null or empty, but got @(@{InputObject=ogv -> Out-GridView; SideIndicator=<=}, @{InputObject=shcm -> Show-Command; SideIndicator=<=}).
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\DefaultCommands.Tests.ps1: line 577
577:         $result | Should -BeNullOrEmpty
```
Verify approved aliases list.All approved Cmdlets present (no new Cmdlets added, no Cmdlets removed)
```powershell
Expected $null or empty, but got @('Out-GridView', 'Show-Command').
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\DefaultCommands.Tests.ps1: line 584
584:         $result | Should -BeNullOrEmpty
```
Verify approved aliases list.All present Cmdlets should have the correct ConfirmImpact
```powershell
Expected $null or empty, but got @(@{Name=Write-Verbose; ConfirmImpact=None; SideIndicator=<=}, @{Name=Write-Warning; ConfirmImpact=None; SideIndicator=<=}).
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\DefaultCommands.Tests.ps1: line 606
606:         $result | Should -BeNullOrEmpty
```